src/workers/notifications/commentReply.ts

```diff
@@ -49,12 +49,7 @@ const worker: NotificationWorker = {
       .distinct(true)
       .where('"parentId" = :parentId', { parentId: parent.id })
       .andWhere('"userId" not in (:...exclude)', {
-        exclude: [
-          parent.userId,
-          comment.userId,
-          postCtx.post.authorId ?? '',
-          postCtx.post.scoutId ?? '',
-        ],
+        exclude: [parent.userId, comment.userId],
```

Comment:
The reason we had them in the list is to prevent multiple notifications for the same reason.
I think authors or scouts receive notifications for any new comment anyway, so please double check that

Reply:
I've tested but you still receive only one notification because on `articleNewCommentHandler` (from `src/workers/notifications/utils.ts`) you unsubscribe onece you comment. There is this piece of code:

```ts
  if (isReply && (post.authorId || post.scoutId)) {
    const ids = [...new Set([post.authorId, post.scoutId])];
    const threadFollower = await repo
      .createQueryBuilder()
      .select('"userId"')
      .where(`(id = :id OR "parentId" = :id)`, { id: comment.parentId })
      .andWhere({ userId: In(ids) })
      .groupBy('"userId"')
      .getRawMany();

    if (threadFollower.length) {
      threadFollower.forEach(({ userId }) => excludedUsers.push(userId));
    }
  }
```

I recently tried this locally with 3 users and noticed that, as an author, I stopped receiving notifications only once I (as the author) commented on the post. I don't know if the fix below is the best solution, but something is conflicting between these two handlers.

Reply:
I'll take a look next week, tnx for checking

Reply:
Basically above logic @ilasw mentioned is to avoid including any users following a thread from getting new comment notification since they would get reply notification. 

Looks to me its safe to remove scout and author from array in commentReply as was done here because if they commented in the thread they probably want a notification for reply cc: @idoshamun ?

Locally it does not look there are any double notifications from what I can see.

Reply:
@idoshamun is ok to merge?

Reply:
Ok! Go ahead :)